README.md

```diff
@@ -12,22 +12,83 @@
 
 ---
 
-Theme Kit is a cross-platform tool for building Shopify Themes. Theme Kit is a single binary that has no dependencies.
+Theme Kit is a cross-platform command line tool that you can use to build Shopify themes.
 
 ## Features
-- Upload Themes to Multiple Environments
-- Fast Uploads and Downloads
-- Watch for local changes and upload automatically to Shopify
-- Works on Windows, Linux and macOS
 
-## Installation
+With Theme Kit, you can use your own development tools to interact with the Shopify platform in the following ways:
 
-You can find installation instructions for each platform on the [Docs Website](https://shopify.github.io/themekit/#installation)
+- Use workflow tools like Git to work with a team of theme developers.
+- Upload themes to multiple environments.
+- Watch for local changes and upload them automatically to Shopify.
+- Work on Linux, macOS, and Windows.
 
-## Setup, Usage and Commands
+## Install Theme Kit
 
-Please find further usage instructions on the [Theme Kit website](https://shopify.github.io/themekit/)
+You can install Theme Kit using the command line on the following operating systems:
 
-## Contribution & Development
+- [Linux](https://shopify.dev/tools/theme-kit/getting-started#linux)
+- [macOS](https://shopify.dev/tools/theme-kit/getting-started#macos)
+- [Windows](https://shopify.dev/tools/theme-kit/getting-started#windows)
 
-Please see the [contributing guidlines](https://github.com/Shopify/themekit/blob/master/CONTRIBUTING.md)
+### Install Theme Kit manually
```

Comment:
Should we make sure they only do this if they know what they are doing? Otherwise we should be redirecting them to the shopify.dev site for package manager installation. I am worried that this takes up too much space and developers will start to think that this is how you install themekit in general.

Lets take out the table with the binary downloads because I think we will move them to the github releases, and maybe move the manual installation instructions into a smaller md file that is off the beaten path.

Also can the AUR package not live with the rest of shopify.dev installation instructions?

Reply:
Thanks, @tanema! Great feedback. I've lifted the manual installation instructions out of the main README now, created a new `MANUAL_INSTALL.md` file to contain that content, removed the table with the binary downloads, and linked out to the GitHub releases page. I've also included a note to redirect them to the command line installation instructions on shopify.dev (to further encourage the preferred / recommended installation method).

In terms of the `AUR` package, I think we want to provide [the green path](https://github.com/Shopify/shopify-dev/pull/5070#discussion_r565603966) on shopify.dev. Any alternate methods, including manual installation steps, and accessing Theme Kit through community packages, can probably remain in the new `MANUAL_INSTALL.md` topic in the GitHub repo.